apps/system/js/dialer_agent.js

```diff
@@ -200,8 +173,8 @@
     }
 
     var self = this;
-
-    self._startAlerting();
+    // Silence the ringtone if more than one call is present
+    this._startAlerting(/* silent */ (this.numOpenLines() > 1));
```

Comment:
This is happening only with two calls (one connected one incoming) and they're both sharing the same ```Audio``` object contained in ```this._player```